fun a(x,y){
    print x+y;
}

a(1,2); // ASSERT 3


a(11,1,3); // assert_error Expected 1 arguments but got 2.